schemas/telemetry/frecency-update/frecency-update.1.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "client_id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "payload": {
+      "properties": {
+        "loss": {
+          "type": "number"
+        },
+        "model_version": {
+          "type": "number"
```

Comment:
Out of curiosity, how often does the model version change and what is its purpose?

Reply:
The version tells the server what iteration the client based the update on. Every 30 minutes a new iteration is started. To apply the model updates, we need to know if the client computed them using the latest model version.